src/cargo/ops/cargo_new.rs

```diff
@@ -17,11 +17,13 @@ use toml;
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub enum VersionControl { Git, Hg, NoVcs }
 
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Ideally these extra derive modes wouldn't be necessary, how come they were added?